browser/lib/util/defaults.js

```diff
@@ -15,11 +15,22 @@ var Defaults = {
 	transports:               ['web_socket', 'xhr', 'iframe', 'jsonp']
 };
 
+/* If an environment option is provided, the environment is prefixed to the domain
+   i.e. rest.ably.io with environment sandbox becomes sandbox-rest.ably.io */
+Defaults.environmentHost = function(environment, host) {
+	if (!environment || (String(environment).toLowerCase() === 'production')) {
```

Comment:
Is there a case where environment is not a string? You already check for the presence of environment (not being falsy), so I would not think you need to convert it to a string, do you?